media-plugins/live/live-2020.03.06.ebuild

```diff
@@ -76,7 +76,7 @@ src_configure() {
 
 src_compile() {
 	export suffix="${LIVE_ABI_VERSION}.so"
-	local link_opts="$(usex ssl '-lssl' '') -L. ${LDFLAGS}"
+	local link_opts="$(usex ssl '-lssl -lcrypto' '') -L. ${LDFLAGS}"
```

Comment:
I wonder if it wouldn't be better to utilize pkg-config (something like `pkg-config --libs libcrypto`) to make sure to get proper libs.